### PR TITLE
Upgrade to terraform 0.12.x

### PR DESCRIPTION
**Pull request by confusdcodr**

### CODE REVIEW

File: .dependabot/config.yml [hunk 1]
```diff
@@ -5,3 +5,15 @@ update_configs:
   - package_manager: "terraform"
     directory: "/"
     update_schedule: "daily"
+
```
Comment (userhas404d) [inline .dependabot/config.yml]: we should probably add `go.dep` and `go.sum` here too
```suggestion

  - package_manager: "go:dep"
    directory: "/tests"
    update_schedule: "daily"

  - package_manager: "go:modules"
    directory: "/tests"
    update_schedule: "daily"
```

Comment (confusdcodr) [reply .dependabot/config.yml]: Fixed

Comment (lorengordon) [reply .dependabot/config.yml]: We're not actually using `go:dep`, that's the one we replaced with go modules. Doesn't hurt to be here, just fyi.

Comment (confusdcodr) [reply .dependabot/config.yml]: I only added the configuration for `go:modules`

**Pull request merged**